clients/ui/bff/data/registered_model.go

```diff
@@ -6,6 +6,7 @@ import (
 	"fmt"
 	"github.com/kubeflow/model-registry/pkg/openapi"
 	"github.com/kubeflow/model-registry/ui/bff/integrations"
+	"net/url"
```

Comment:
Not about this line, but L13:

As you are going to touch the code with the review, I wonder if we should change this for GetAllRegisteredModels